src/Selenium2Driver.php

```diff
@@ -953,7 +953,7 @@ public function mouseOver(string $xpath)
     private function scrollElementIntoView(Element $element): void {
         $script = <<<JS
             var e = arguments[0];
-            e.scrollIntoView({ behavior: 'instant', block: 'end', inline: 'nearest' });
+            e.scrollIntoView({ behavior: 'instant', block: 'center', inline: 'nearest' });
```

Comment:
Thinking about this change. I'm wondering why it is necessary?

Reply:
I tried the fix from https://github.com/Lullabot/MinkSelenium2Driver/pull/15 but it didn't solve the issue.
I'll spend some more time shortly to try to understand what's going on but I'm not sure the issue is specific to my project, I think it's an edge case (with size attributes?).

Reply:
@vever001 
#15 does not scroll for you. it only prevents the re-scrolling. If you apply #15, you would need to scroll to the element before making an assertion.

Reply:
We faced similar issue after upgrading Drupal 10.2 to 10.3, where `behat/mink-selenium2-driver` got replaced with this library. The tests started to fail and patching the `e.scrollIntoView()` block argument to `center` fixed the issues of layouts with floating toolbar receiving the press/click events.